tests/dummy/app/models/company.js

```diff
@@ -1,16 +1,18 @@
-import Model from 'ember-data/model';
-import { attr } from '@ember-decorators/data';
+import DS from 'ember-data';
 import {
   hasValidations,
   validator,
   buildValidations
 } from 'ember-cp-validations';
 
+const { attr, Model } = DS;
+
 export const Validations = buildValidations({
   name: validator('presence', { presence: true, description: 'Name' })
 });
 
+export default
```

Comment:
Should be 

```js
@hasValidations(Validations)
export default class Company extends Model {}
```

I believe this is why the tests are failing.

Reply:
Fixed this but tests are still failing